packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -48,3 +66,124 @@ class MaterialLocalizations {
     return Localizations.of<MaterialLocalizations>(context, MaterialLocalizations);
   }
 }
+
+/// Defines text geometries for the three language categories defined in
+/// https://material.io/guidelines/style/typography.html.
+class _MaterialTextGeometry {
```

Comment:
these should be public since other packages that implement localisations will need to be able to refer to them.